R/use_beachmat.R

```diff
@@ -0,0 +1,37 @@
+#' Use beachmat
+#'
+#' Creates \code{src/} and adds needed packages to \code{DESCRIPTION}.
+#'
+#' For more details see the 'Linking' vignette, available by running
+#' \code{vignette("linking", package = "beachmat")}.
+#' @importFrom usethis use_directory use_git_ignore
+#' @export
```

Comment:
I'd prefer this not to be exported; it doesn't need to be run in dependent packages every time. If you must export it; I'm not using the Roxygen documentation tools, so you'll have to actually write a manpage manually, i.e., just delete the stuff here and remove the "Do not edit by hand" comments in the manpage.

Reply:
No longer exported although manpage is retained (manual).

Reply:
Also no longer importing anything from **usethis** into `NAMESPACE`